rdr_client/hpo_assigner.py

```diff
@@ -2,7 +2,7 @@
 
 Usage:
   ./run_client.sh --project all-of-us-rdr-prod --account $USER@pmi-ops.org \
-      hpo_assigner.py participant_ids_and_hpos.csv [--dry_run]
+      hpo_assigner.py participant_ids_and_hpos.csv --pairing [site|organization|awardee] [--dry_run]
```

Comment:
This works OK assuming that we are going to get pairing files that look like 

<PARTICIPANT ID>,<AWARDEE/ORGANIZATION/SITE ID>

and that all of the entries have the same level of hierarchy.

That seems OK to me, but we should first check in with Ashley to make sure that ServiceNow is going to start sending us files that look like that.... and that they will actually change to use this.

Reply:
(Let's hold off on this PR until we hear back about this.)

Reply:
spoke with the team, this technique is good.

Reply:
So is ServiceNow going to start sending us data at org / site level?

Reply:
according to Ashley, yes.